release-notes/swan-lake-beta4-release-note.md

```diff
@@ -298,6 +298,20 @@ float f13 = 0x1A.0;
 float f14 = 0x1A.0p4;
 ```
 
+- Intervening white spaces are disallowed in the qualified identifier to avoid parsing ambiguity between ternary conditional expression and qualified identifier.
+  
+```ballerina
+import ballerina/io;
+
+public function main() {
+io:print("Ballerina"); // valid
+
+io : print("Ballerina"); // compilation error: intervening whitespaces are not allowed in qualified identifier
+}
+```
+
+With this `x ? a : b:c` will be parsed as `x ? a : (b:c)` since colon with spaces is interpreted only as part of a conditional expression.
```

Comment:
```suggestion
With this, `x ? a : b:c` will be parsed now as `x ? a : (b:c)` since the colon with spaces is interpreted only as part of a conditional expression.
```